data/data-pipeline/data_pipeline/content/config/generate_data_dictionary.py

```diff
@@ -0,0 +1,179 @@
+"""
+Quick script to generate a data dictionary for the download packet.
+
+A few notes:
+(1) This will ONLY crosswalk the downloadable CSV and the downloadable EXCEL file.
```

Comment:
If we want to make a shp dictionary (beyond -- "here's the column and what it means in plain English") we will need to do it separately since the shapefiles have more fields.